src/js/wrap.cpp

```diff
@@ -358,28 +477,31 @@ namespace ccf::js
   static JSValue js_kv_map_clear(
     JSContext* ctx, JSValueConst this_val, int argc, JSValueConst* argv)
   {
-    auto handle = static_cast<KVMap::Handle*>(
-      JS_GetOpaque(this_val, kv_map_handle_class_id));
+    js::Context& jsctx = *(js::Context*)JS_GetContextOpaque(ctx);
 
     if (argc != 0)
     {
       return JS_ThrowTypeError(
         ctx, "Passed %d arguments, but expected 0", argc);
     }
 
+    auto handle = _get_map_handle(jsctx, this_val);
+    if (handle == nullptr)
```

Comment:
This seems to happen a lot, perhaps there's a JS_CHECK_HANDLE(), to go with the other JS_CHECK*() macros.